components/widgets/admin_console/admin_panel.scss

```diff
@@ -6,7 +6,7 @@
 .AdminPanel {
     @include border-radius($border-rad);
     @include box-shadow(0 1px 2px 0 alpha-color($black, .5));
-    @include clearfix;
+
```

Comment:
This is replaced with the `clearfix` class from bootstrap being added to the element itself